.yarn/sdks/typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "typescript",
-  "version": "4.4.3-sdk",
+  "version": "4.5.4-sdk",
```

Comment:
@kaydelaney i think this is the typescript that is used in vscode etc - is this the only change that needs making to upgrade the version? 🤔

Reply:
I think so? Not 100% sure to be honest 😅
Let me check

Reply:
Pulled the PR and tested it, seems to work fine 👍